src/main/java/ether/Settings.java

```diff
@@ -60,8 +64,9 @@
         try {
             Object obj = parser.parse(new FileReader("rm-ethereum.conf"));
             JSONObject jsonObject = (JSONObject) obj;
-            appendingZerosForETH = ((String) jsonObject.get("appendingZerosForETH"));
-            appendingZerosForTKN = ((String) jsonObject.get("appendingZerosForTKN"));
+
+            //appendingZerosForETH = ((String) jsonObject.get("appendingZerosForETH"));
+            //appendingZerosForTKN = ((String) jsonObject.get("appendingZerosForTKN"));
```

Comment:
Can these just be deleted or do they need to stick around?

Reply:
Uncommented, maybe we will need for testing.